packages/core-typings/src/import/IImportFileData.ts

```diff
@@ -1,8 +1,10 @@
 import type { IImportChannel } from './IImportChannel';
+import type { IImportMessage } from './IImportMessage';
 import type { IImportUser } from './IImportUser';
 
 export interface IImportFileData {
 	users: Array<IImportUser>;
 	channels: Array<IImportChannel>;
 	message_count: number;
+	messages: Array<Pick<IImportMessage, '_id' | 'rid' | 'u'>>;
```

Comment:
we cannot store a list of messages in a single record like this.. it doesn't scale.. MongoDB maximum object size is 16MB, it seems a lot, but if you're importing big channels this limit can be reached..